frontend/src/components/common/Resource/MainInfoSection/MainInfoSectionHeader.tsx

```diff
@@ -118,7 +118,7 @@ export function MainInfoHeader<T extends KubeObject>(props: MainInfoHeaderProps<
 
   return (
     <SectionHeader
-      title={title || (resource ? resource.kind : '')}
+      title={title || (resource ? `${resource.kind} (${resource.getName()})` : '')}
```

Comment:
I think we can make it more readable by using `KIND: NAME` than the `()`, WDYT? Also saves 1 character 😉 
Example:
![Screenshot with example](https://github.com/user-attachments/assets/4b2fa116-25be-49ea-8589-dd46f97a6fea)

Reply:
Looks great. I changed it to this way.